src/run/handlers/cache.cts

```diff
@@ -492,7 +465,8 @@ export class NetlifyCacheHandler implements CacheHandlerForMultipleVersions {
   }
 
   resetRequestCache() {
```

Comment:
Now that it's a no-op, are we free to remove this function or does next still call this?